examples/reverse-proxy/MNIST_using_Reverse_Proxy.md

```diff
@@ -0,0 +1,66 @@
+
+This example uses one training batch and one test batch. Both batch files are stored in an archive file called `mnist.npz`. The Machine Learning program, after conversion to Swarm Learning for the TensorFlow-based Keras platform, is in `examples/reverse-proxy/mnist/model` and the TensorFlow-based file name is `mnist tf.py`.
+
+This example shows the Swarm training of MNIST model using two ML nodes. ML nodes are automatically spawned by SWOP node. Swarm training is initiated by SWCI node and orchestrated by a SN node. This example also shows how private data and shared model can be mounted to ML nodes for Swarm training. For more information, see the profile files and task definition files placed under `examples/reverse-proxy/mnist/swop` and `examples/reverse-proxy/mnist/swci` folders, respectively.
+
+## Running the MNIST example
+
+1.  On host-1, navigate to swarm-learning folder.
+
+    ``` {#CODEBLOCK_MYT_33W_NVB}
+    cd swarm-learning
+    ```
+
+2.  Run the `run-all` script from the `swarm-learning` folder with `APLS IP` argument and `wait time for sn`.
+
+    ``` {#CODEBLOCK_LQT_DNY_NVB}
+    .\examples\reverse-proxy\run-all 172.1.1.1 4m
+    ```
+
+    First argument `APLS IP` in this command is a mandatory parameter. Second argument `wait time for sn` if not passed, by default it will wait for 5 minutes. This `wait time for sn` argument is similar to sleep, where `m` denotes minutes.
+
+    This script will take care of starting BIND9 container, NGINX container and rest all of the swarm containers in the sequential manner. All the run-script commands will now take FQDN's as service parameter arguments instead of ports.
+
+    **NOTE:** SN-P2P-PORT still needs 30303 port in the host machine, where SN container runs.
+
+3.  On host-1, Swarm training is automatically started when the run task \(`swarm_mnist_task`\) gets assigned and executed. Open a new terminal on host-1 and monitor the docker logs of ML nodes for Swarm training. Swarm training will end with the following log message.
+
+    ``` {#CODEBLOCK_T4Y_2JW_NVB}
+    SwarmCallback : INFO : All peers and Swarm training rounds finished. Final Swarm model was loaded.
+    ```
+
+    The final Swarm model is saved inside `workspace/reverse-proxy/mnist/model` directory on host-1. All the dynamically spawned SL and ML nodes exits after Swarm training. The SN and SWOP nodes continue to run.
+
+4.  On host-1, to clean-up, run the `scripts/bin/stop-swarm` script to stop and remove the swarm container nodes of the previous run. If required, backup the container logs. This example builds and starts BIND9 and NGINX, so remove their respective images and containers. Then, remove docker volume \(`sl-cli-lib`\) and delete the `workspace` directory.
+
```

Comment:
Please compare this file against "[README.md](https://github.hpe.com/hpe/swarm-learning-docs/tree/master/examples/reverse-proxy/mnist#readme)" and apply changes. 
1. 2 more References heading missing
2. Square braces for numbers